src/data/python-quiz.ts

```diff
@@ -1284,6 +1284,15 @@ const pythonQuiz = [
     Explanation: "Since booleans in Python are seen as numeric types, you can perform basic math operations on them. True would equate to 1 while False would equate to 0.",
     Link: "https://www.freecodecamp.org/news/truthy-and-falsy-values-in-python/",   
   },
+  {
+    Question: "In Python, if words = ['Welcome', 'To', 'FreeCodeCamp'], then what will be the result returned when the following code is executed: print(list(map(len, words)))?",
```

Comment:
We need to be careful with long questions, because it doesn't render well on the page. 

<img width="1476" alt="Screen Shot 2022-10-11 at 10 09 45 PM" src="https://user-images.githubusercontent.com/67210629/195255224-07a95933-190a-4790-a024-c86b53bc1545.png">

```suggestion
    Question: "In Python, what is the return value: print(list(map(len, ['Welcome', 'To', 'FreeCodeCamp'])))?",
```